src/main/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImpl.java

```diff
@@ -81,9 +82,15 @@ public void runAutomatedTask(Integer taskId) {
     public DetailedAutomatedTask updateAutomatedTask(Integer taskId, AutomatedTaskPatch automatedTaskPatch) {
         var automatedTask = getAutomatedTaskEntityById(taskId);
 
-        automatedTask.setTaskEnabled(automatedTaskPatch.getIsActive());
+        if (automatedTaskPatch.getIsActive() != null) {
+            automatedTask.setTaskEnabled(automatedTaskPatch.getIsActive());
+            auditApi.record(ENABLE_DISABLE_JOB);
+        }
 
-        auditApi.record(ENABLE_DISABLE_JOB);
+        if (automatedTaskPatch.getBatchSize() != null) {
```

Comment:
if batch size is null should we set the batch size to be null in the automated task?